plugins/woocommerce/src/StoreApi/Routes/V1/Checkout.php

```diff
@@ -547,13 +552,23 @@ private function process_customer( \WP_REST_Request $request ) {
 				case 'registration-error-invalid-email':
 					throw new RouteException(
 						'registration-error-invalid-email',
-						__( 'Please provide a valid email address.', 'woocommerce' ),
+						esc_html__( 'Please provide a valid email address.', 'woocommerce' ),
 						400
 					);
 				case 'registration-error-email-exists':
 					throw new RouteException(
 						'registration-error-email-exists',
-						__( 'An account is already registered with your email address. Please log in before proceeding.', 'woocommerce' ),
+						sprintf(
+							// Translators: %s Email address.
+							esc_html__( 'An account is already registered with %s. Please log in or use a different email address.', 'woocommerce' ),
```

Comment:
Wandering a bit  off topic but, since we're changing this translation, wouldn't this be a good opportunity to remove the chance of scanning for registered emails this way? 🤔

Reply:
The WP login page reveals this also. So I'd rather tackle separately if at all.

Reply:
Cool 👍🏼